### PR TITLE
Use windows-compatible --workspace_status_command

### PR DESCRIPTION
**Pull request by filipesilva**
Related to https://github.com/bazelbuild/bazel/issues/5958

### CODE REVIEW

**Comment by filipesilva** [comment]
Seems like the circleci containers we use don't have node so a node script won't work.

I think for cross platform scripts we'll have to go down to some available runtime and node seems a good candidate since this is a node-based project.

@alexeagle @gregmagolan WDYT?

**Comment by filipesilva** [comment]
@alexeagle done, but PTAL to see if it matches what you expect.

File: README.md [hunk 1]
```diff
@@ -340,6 +340,7 @@ rule is that minors are breaking changes and patches are new features).
 1. `git commit -a -m 'Update docs for release'`
 1. `npm config set tag-version-prefix ''`
 1. `npm version minor -m 'rel: %s'` (replace `minor` with `patch` if no breaking changes)
-1. Build npm packages and publish them: `bazel run //internal:npm_package.publish && bazel run //internal/karma:npm_package.publish`
+1. Create a clean directory to use as output: `rm -rf ./rules_typescript_release && mkdir ./rules_typescript_release`
+1. Build npm packages and publish them: `bazel run //internal:npm_package.publish --config=release --output_base=rules_typescript_release && bazel run //internal/karma:npm_package.publish --config=release --output_base=rules_typescript_release`
```
Comment (filipesilva) [inline README.md]: I assume the run command will run the build, since release instructions didn't actually mention building.

**Comment by kyliau** [comment]
@filipesilva Could you please rebase?

**Comment by filipesilva** [comment]
@kyliau done.

File: DEVELOPING.md [hunk 2]
```diff
@@ -62,7 +62,8 @@ rule is that minors are breaking changes and patches are new features).
 1. `git commit -a -m 'Update docs for release'`
 1. `npm config set tag-version-prefix ''`
 1. `npm version minor -m 'rel: %s'` (replace `minor` with `patch` if no breaking changes)
-1. Build npm packages and publish them: `bazel run //internal:npm_package.publish && bazel run //internal/karma:npm_package.publish`
+1. Create a clean directory to use as output: `rm -rf ./rules_typescript_release && mkdir ./rules_typescript_release`
+1. Build npm packages and publish them: `bazel run //internal:npm_package.publish --config=release --output_base=rules_typescript_release && bazel run //internal/karma:npm_package.publish --config=release --output_base=rules_typescript_release`
 1. `bazel build :release`
```
Comment (filipesilva) [inline DEVELOPING.md]: @alexeagle @mgechev this step wasn't here when I looked at the instructions before, and I am not sure if it affects/is affected by the steps I'm adding. Can you have a look please?

**Comment by alexeagle** [comment]
We are going to stop publishing releases from this repo

**Pull request closed without merging**